app/assets/javascripts/discourse/app/initializers/post-decorations.js

```diff
@@ -103,6 +105,31 @@ export default {
         },
         { id: "onebox-source-icons" }
       );
+
+      api.decorateCookedElement(
+        (element) => {
+          element
+            .querySelectorAll(".video-container")
+            .forEach((videoContainer) => {
+              const video = videoContainer.getElementsByTagName("video")[0];
+              video.addEventListener("loadeddata", () => {
+                later(() => {
+                  if (video.videoWidth === 0 && video.videoHeight === 0) {
```

Comment:
I think it's safe to assume if either is 0, there's no video?

```suggestion
                  if (video.videoWidth === 0 || video.videoHeight === 0) {
```